apps/web/src/services/administration/iam/user/modules/user-management-modal/UserManagementModal.vue

```diff
@@ -205,7 +205,14 @@ export default {
                 password: formState.password || '',
             };
             if (userPageState.visibleCreateModal) {
-                data.backend = formState.activeTab === 'local' ? 'LOCAL' : 'EXTERNAL';
+                if (formState.activeTab === 'local' || formState.activeTab === 'apiOnly') {
+                    data.backend = 'LOCAL';
+                    if (formState.activeTab === 'apiOnly') {
+                        data.user_type = 'API_USER';
+                    }
+                } else {
+                    data.backend = 'EXTERNAL';
+                }
```

Comment:
nitpick: I think we can make it more explicit. :)

```suggestion
                if (formState.activeTab === 'local') {
                    data.backend = 'LOCAL';
                }  else if (formState.activeTab === 'apiOnly') {
                        data.backend = 'LOCAL';
                        data.user_type = 'API_USER';
                } else {
                    data.backend = 'EXTERNAL';
                }
```